apps/web/src/utils/trpc/server.ts

```diff
@@ -1,14 +1,48 @@
 import { env } from "@/env"
 import type { AppRouter } from "@dotkomonline/gateway-trpc"
 import * as trpc from "@trpc/client"
+import { getToken } from "next-auth/jwt"
+import type { RequestCookies } from "next/dist/compiled/@edge-runtime/cookies"
+import type { ReadonlyHeaders } from "next/dist/server/web/spec-extension/adapters/headers"
+import type { ReadonlyRequestCookies } from "next/dist/server/web/spec-extension/adapters/request-cookies"
```

Comment:
these paths are a little sus but seems ok for now. hopefully we can get rid of all of this with next auth 5